docs/StardustDocs/topics/ColumnSelectors.md

```diff
@@ -251,18 +250,18 @@ df.select { take(2) and col(3) }
 
 ```kotlin
 // first/last n columns in column set
-df.select { allDfs().take(3) }
-df.select { allDfs().takeLast(3) }
+df.select { allRec(includeGroups = false).take(3) }
```

Comment:
hm, i see various different options in this PR. Sometimes it's all().recursively(), sometimes all().rec(), and this. I saw that you did it intentionally, but why?

Reply:
Yes, it does exactly the same, similar to allDfs and all().dfs. First I decided to use all().rec(ursively) everywhere, but indeed for completion I decided to have allRec(ursively) present too. I think we can just keep all options.